.gitlab-ci.yml

```
@@ -56,6 +56,13 @@ flake8-diff:
     - apk add --update --no-cache git
     - git diff -U0 origin/master | flake8 --verbose --diff
 
+shellcheck:
+  stage: build
+  image: pipelinecomponents/shellcheck:0.5.4
+  script:
+    - apk add --update --no-cache git
+    - git ls-files --ignore --exclude '*.sh' | xargs shellcheck -x -P SCRIPTDIR
```

Please use the -z option and xargs -0, to support spaces if we ever need to add those.

The option is called --ignored, not --ignore. And what you're doing there is crazy: you ask to show ignored files and give an additional ignore pattern of *.sh, which indeed gives you all the *.sh files (ignored or not).

The pattern doesn't capture prplmesh_utils.sh.in.

So:
```suggestion
    - git ls-files -z '*.sh' '*.sh.in' | xargs -0 shellcheck -x -P SCRIPTDIR
```

---

Applied!